src/sbvr-api/translations.ts

```diff
@@ -208,6 +208,11 @@ export const translateAbstractSqlModel = (
 			const { $toResource, ...definition } = translationDefinition;
 			const hasToResource = typeof $toResource === 'string';
 			if (hasToResource) {
+				if ($toResource.includes('$')) {
+					throw new Error(
+						'`$toResource` should be the plain name of the resource in the subsequent model and not be targeting a specific model',
```

Comment:
+1 on adding a check that warns/errors when defining $toResource is not necessary: eg
```
device: {
    $toResource: `device` // or `device$${toVersion}`,
    // ...
},
```

Reply:
Wrt allowing $toResource to use a different version, I would atm prefer to allow that, at least until we get @otaviojacobi 's PR in (or any fix for the underlying issue), so that we don't block v7.
See: https://github.com/balena-io/pinejs/pull/781